tests/codecs/test_datetime.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+
+from datetime import datetime
+
+from mlserver.codecs import DatetimeCodec
+from mlserver.types import RequestInput, ResponseOutput
+
+TestDatetimeIso = "2021-08-24T15:01:19"
```

Comment:
do we care about timezone aware testing as well?

Reply:
We don't care that much, in the sense that `datetime` should be able to parse the timezone if it's present on the string. But I'll add a test to make sure that this actually happens.